docs/source/development/traffic_router/traffic_router_api.rst

```diff
@@ -41,32 +41,32 @@ Request Structure
 
 	GET /crs/stats HTTP/1.1
 	Host: trafficrouter.infra.ciab.test
-	User-Agent: curl/7.47.0
+	User-Agent: curl/7.52.1
 	Accept: */*
 
 Response Structure
 ------------------
 .. code-block:: http
-	:caption: Response Example
+	:caption: Response Example (JSON expanded)
```

Comment:
You don't need to specify that the JSON has been expanded. Users are instructed to ignore the value of any `Content-Length` header.

Reply:
Resolved in f682425f.